src/dune_pkg/local_package.mli

```diff
@@ -0,0 +1,13 @@
+open! Import
```

Comment:
I like the PR description as it adds context and reasoning for this, so I think adding a documentation comment to this module what a `Local_package` is and why it exists would be good!

Reply:
Will add in a later change.